syntaxes/v.tmLanguage.json

```diff
@@ -1024,6 +1023,14 @@
 					"name": "constant.other.placeholder.v"
 				}
 			]
+		},
+		"illegal-name": {
+			"patterns": [
+				{
+					"match": "\\d\\w+",
+					"name": "invalid.illegal.v"
+				}
+			]
```

Comment:
This can be simplified to just:

~~~json
		"illegal-name": {
			"match": "\\d\\w+",
			"name": "invalid.illegal.v"
		}
~~~

Reply:
Updated.